src/Components/Header.jsx

```diff
@@ -1,23 +1,38 @@
-import React from 'react'
-import { Link } from "react-router-dom";
-import { withRouter } from "react-router";
-const Header = () => {
+import { Button, Grid, Typography, makeStyles } from "@material-ui/core"
+import React from "react"
+
+const useStyles = makeStyles({
+  btn: {
+    textAlign: "right",
+    paddingRight: 20,
+  },
+  heading: {
+    paddingLeft: 20,
+  },
+})
 
+const Header = () => {
+  const classes = useStyles()
+  return (
+    <Grid container direction="row" alignItems="center">
+      <Grid item xs={6}>
+        <Typography
+          className={classes.heading}
+          variant="h4"
+          color="textSecondary"
+        >
+          {" "}
+          ToDo List
+        </Typography>
+      </Grid>
 
-    return (
-          <div className="container">
-            <div className="row">
-              <div className="col-md-10 ">
-                <h1 className="header text-right"> Asim's ToDo List</h1>
-              </div>
-                <div className="col-md-2 pt-5">
-                <Link to="/" className="btn btn-danger">
-                  Logout
-                </Link>
-              </div>
-            </div>
-          </div>
-    );
+      <Grid className={classes.btn} item xs={6}>
+        <Button href="/" variant="contained" size="small" color="secondary">
```

Comment:
use routing

Reply:
Wrapped Button inside Link Component for routing